lib/Conversion/StandardToLLVM/ConvertStandardToLLVM.cpp

```diff
@@ -1026,6 +1026,58 @@ struct DeallocOpLowering : public LLVMLegalizationPattern<DeallocOp> {
   }
 };
 
+// A `tanh` is converted into a call to the `tanh` function.
+struct TanhOpLowering : public LLVMLegalizationPattern<TanhOp> {
+  using LLVMLegalizationPattern<TanhOp>::LLVMLegalizationPattern;
+
+  PatternMatchResult
+  matchAndRewrite(Operation *op, ArrayRef<Value *> operands,
+                  ConversionPatternRewriter &rewriter) const override {
+
+    using LLVMFuncOpT = LLVM::LLVMFuncOp;
+    using LLVMTypeT = LLVM::LLVMType;
+
+    assert(operands.size() == 1 && "tanh takes one operand");
```

Comment:
This should be handled by the verifier, we don't duplicate invariant checks in general.
(Alex is fixing it on integration, just saying FYI for future patches)